doc/api/fs.md

```diff
@@ -102,6 +102,11 @@ example `fs.readdirSync('c:\\')` can potentially return a different result than
 `fs.readdirSync('c:')`. For more information, see
 [this MSDN page][MSDN-Rel-Path].
 
+*Note:* On Windows opening existing hidden file using the `w` flag (either
+through `fs.open` or `fs.writeFile`) will fail with `EPERM`. You can work
```

Comment:
A nit: `fs.open()`, `fs.writeFile()` and `fs.ftruncate()`?

Reply:
Should be either `opening an existing hidden file` or `opening existing hidden files`.

Reply:
Please avoid the use of informal pronouns like `you` in the docs